sections/semantics-interactive-elements.include

```diff
@@ -1,5 +1,23 @@
 <section>
 
+<!--
+
+    Interactive Elements
+
+  This source produces section 4.7: Embedded Content
```

Comment:
I think this header is incorrect. Appears to be a duplicate of the one in semantics-embedded-content.

Reply:
thanks!